ui-components/src/actions/LinkButton.tsx

```diff
@@ -3,10 +3,14 @@ import "./Button.scss"
 import { buttonClassesForProps, buttonInner, ButtonProps } from "./Button"
 import { NavigationContext } from "../config/NavigationContext"
 import { isExternalLink } from "../helpers/links"
+import { faUpRightFromSquare } from "@fortawesome/free-solid-svg-icons"
+import { Icon } from "../icons/Icon"
 
 export interface LinkButtonProps extends Omit<ButtonProps, "onClick"> {
   href: string
   dataTestId?: string
+  newTab?: boolean
+  newTabIcon?: boolean
```

Comment:
Is that make sense to create a more generic prop like just `icon` and control `area-label` by props?

Reply:
Not 100% sure I'm following sorry! The first prop `newTab` is for whether or not the link opens in a new tab, and the `newTabIcon` is for whether or not we should show the newTab icon, and we want both states to be possible.

Reply:
I do like the idea of the icon being standardized, but the Button component (which supplies the `buttonInner` function) already has a mechanism for icons, so I'm confused that there's an icon being rendered within LinkButton separately. Is there a way to pass the icon along with the props to `buttonInner`?